modules/core/src/lib/log.js

```diff
@@ -373,8 +373,12 @@ function normalizePriority(priority) {
 export function normalizeArguments(opts) {
   const {priority, message} = opts;
   opts.priority = normalizePriority(priority);
-
-  const args = opts.args || [];
+  // args should only contain arguments that appear after `message`
```

Comment:
As far as I can tell, this arguments array processing only happens if logging is actually active...  looks good.